### ISSUE TITLE
Error when import dependencies youtube_player: ^3.0.1

### ISSUE THREAD

**Issue by docaohuynh** [sarbagyastha/youtube_player#17]
I have got error when run app when add youtube_player: ^3.0.1 

```
Launching lib/main.dart on Android SDK built for x86 in debug mode...
/MyData/myProject/OwnSource/Mobile/MyProject/test_flutter/android/app/src/main/AndroidManifest.xml:17:9-37 Error:
	Attribute application@label value=(test_flutter) from AndroidManifest.xml:17:9-37
	is also present at [com.github.HaarigerHarald:android-youtubeExtractor:master-SNAPSHOT] AndroidManifest.xml:15:9-41 value=(@string/app_name).
	Suggestion: add 'tools:replace="android:label"' to <application> element at AndroidManifest.xml:15:5-38:19 to override.
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute application@label value=(test_flutter) from AndroidManifest.xml:17:9-37
  	is also present at [com.github.HaarigerHarald:android-youtubeExtractor:master-SNAPSHOT] AndroidManifest.xml:15:9-41 value=(@string/app_name).
  	Suggestion: add 'tools:replace="android:label"' to <application> element at AndroidManifest.xml:15:5-38:19 to override.
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org

```

**Comment by sarbagyastha** [comment]
@docaohuynh add this line to your manifest file 
> tools:replace="android:label"

as in the [Example](https://github.com/sarbagyastha/youtube_player/blob/master/example/android/app/src/main/AndroidManifest.xml)

**Comment by sarbagyastha** [comment]
@docaohuynh the issue is fixed in **[v3.1.0](https://pub.dartlang.org/packages/youtube_player)**
